src/js/server/reset.js

```diff
@@ -78,6 +78,18 @@ fluid.express.user.reset.handler.checkResetCode = function (that, dataSourceResp
 
 fluid.defaults("fluid.express.user.reset.handler", {
     gradeNames:  ["fluid.express.handler"],
+    digest: "{fluid.express.user.reset.post}.options.digest",
```

Comment:
Rather than repeating this material in each handler and also subcomponent definition, why not factor it, and also bindings to it, into a fluid.express.user.passwordEncryptOptions grade?

Reply:
Good idea.  I made a pair of grades, one that "holds" the options, another that "consumes" them.